src/webui/webapplication.cpp

```diff
@@ -769,6 +771,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
+    QString newLocationPath = request().posts["location"].trimmed();
+
+    // check location exists
+    QFileInfo newLocation(newLocationPath);
+    if (!newLocation.exists()) {
+        qDebug() << "Location" << newLocationPath << "is invalid. Not moving files.";
```

Comment:
I still don't see the point in this message. Can you explain to me the need? 
@qbittorrent/demigods, maybe I pick on @Piccirello, but, IMO, we have to do something in this direction. Our debug output is just a bunch of garbage in which it is impossible to discover something really useful! I think we need to clean it and leave there only the most important messages.

P.S. Log is even more unlikely place for such messages.